From 5c3b005aaad70692a8ac2891b402a636ec3cc9e9 Mon Sep 17 00:00:00 2001
From: Christian Hewitt <christianshewitt@gmail.com>
Date: Thu, 15 Dec 2022 04:27:45 +0000
Subject: [PATCH 103/119] WIP: dt-bindings: arm: amlogic: add support for Ugoos
 AM6 plus

The AM6 plus is a revised version of the AM6 Android box based
on the Amlogic W400 reference board, but now with an S922XJ to
bump CPU speeds (rev C silicon) and add Dolby Vision features.

Signed-off-by: Christian Hewitt <christianshewitt@gmail.com>
---
 Documentation/devicetree/bindings/arm/amlogic.yaml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/arm/amlogic.yaml b/Documentation/devicetree/bindings/arm/amlogic.yaml
index b8b2899ad83b..cc0f1ac5f678 100644
--- a/Documentation/devicetree/bindings/arm/amlogic.yaml
+++ b/Documentation/devicetree/bindings/arm/amlogic.yaml
@@ -184,6 +184,7 @@ properties:
               - hardkernel,odroid-n2-plus
               - khadas,vim3
               - ugoos,am6
+              - ugoos,am6-plus
           - const: amlogic,s922x
           - const: amlogic,g12b
 
-- 
2.34.1

